cucumber_dnieto/features/page_objects/add_single_device_overlay.rb

```diff
@@ -0,0 +1,9 @@
+require 'site_prism'
+
+class AddSingleDeviceOverlay < SitePrism::Page
```

Comment:
should be a Section instead of a Page
turning this class into a section, would not required to set the url below